packages/react-aria-components/docs/Disclosure.mdx

```diff
@@ -103,6 +106,108 @@ import {UNSTABLE_Disclosure as Disclosure, Button, UNSTABLE_DisclosurePanel as D
 
 </details>
 
+
+## Features
+
+Custom styled disclosures can be difficult to build in an accessible way with the correct aria pattern. When implementing, you must manually ensure that the button and panel are semantically connected via ids for accessibility. `Disclosure` helps with this and other accessibility features while allowing for custom styling.
+
+* **Flexible** - Structured such that it can be used standalone or combined with other disclosures to form a `DisclosureGroup`
+* **Accessible** - When focused, a disclosure's visibility can be toggled with either the <Keyboard>Enter</Keyboard> or <Keyboard>Space</Keyboard> key, and the appropriate aria attributes are automatically applied.
+* **Styleable** - Keyboard focus, disabled and expanded states are provided for easy styling. These states only apply when interacting with an appropriate input device, unlike CSS pseudo classes.
+
+## Anatomy
+
+<Anatomy />
+
+A disclosure consists of a button and panel of content. The button is wrapped in a heading element with the appropriate level based on the architecture of the page, and it contains the label representing content within the panel. The panel is the section of content that is associated with the button which is either expanded or collapsed.
+
+```tsx
+import {Button, UNSTABLE_Disclosure as Disclosure, UNSTABLE_DisclosurePanel as DisclosurePanel} from 'react-aria-components';
+
+<Disclosure>
+  <h3>
```

Comment:
Can we use `Heading` here?

Reply:
should we use `Heading` in place of `h3` for the other examples too?

Reply:
On second thought, the heading isn't required by the disclosure pattern, although it is in the accordion pattern. I think if keep it we can use the RAC one for consistency and to have a class name built in.